config/kafka-instance-types-configuration.yaml

```diff
@@ -24,10 +31,29 @@
 #   - quotaConsumed: Quota consumed for selecting this size for a Kafka instance.
 #   - quotaType: Quota type that will be consumed when this size is selected.
 #   - capacityConsumed: Data plane cluster capacity consumed by this Kafka instance size (only used for manual scaling)
+
 ---
 supported_instance_types:
```

Comment:
Please double-check that all the values specified here for all instance types are the correct values